bauble/src/value/mod.rs

```diff
@@ -665,13 +665,15 @@ impl PathKind {
 pub struct DelayedRegister {
     pub asset: Spanned<TypePath>,
     pub reference: Spanned<PathKind>,
+    pub expected_ty_path: Option<Spanned<PathKind>>,
```

Comment:
The type we want a potential reference to resolve into.

Reply:
Good doc comment?